src/index.ts

```diff
@@ -226,10 +262,11 @@ const updateElements = (
   )
 }
 
-export const createHead = () => {
+export const createHead = (options: HeadOptions = {}) => {
```

Comment:
I am a bit hesitant about this options approach. Couldn't we make it dynamic as an option of `useHead`?

Reply:
I took a spin at this in https://github.com/vueuse/head/pull/77 before realizing this PR existed. I can confirm this option is needed. In an Inertia app, the title template would exist in a `main.ts`, not in an SFC. And I also couldn't make SSR work properly without using `createHead`.